test/test_jit.py

```diff
@@ -1158,6 +1158,60 @@ def get_forward(m):
                    .check_next('prim::CallMethod[name="forward"](%observer_for_') \
                    .run(str(m._c._get_method("forward").graph))
 
+    def test_insert_quant_dequant(self):
+        class Observer(torch.nn.Module):
+            def __init__(self):
+                super(Observer, self).__init__()
+
+            def forward(self, x):
+                return x
+
+            @torch.jit.export
+            def calculate_qparams(self):
+                return torch.tensor([2.0]), torch.tensor([3])
+
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(3, 5, 3).float()
+
+            def forward(self, x):
+                return self.conv(x)
+
+        m = torch.jit.script(M())
+        observer = torch.jit.script(Observer())
+        torch._C._jit_pass_constant_propagation(m.graph)
+
+        m._c = torch._C._jit_pass_prepare_quant(m._c, "forward",
+                                                observer._c,
+                                                observer._c)
+        data = torch.randn(1, 3, 10, 10, dtype=torch.float)
+
+        def get_forward(m):
+            return m._c._get_method('forward')
+        get_forward(m)(data)
+
+        # right now the result will have extra observer modules
+        # will fix later when we figure out how to remove modules
+        m._c = torch._C._jit_pass_insert_quant_dequant(m._c, "forward")
+
+        get_forward(m)(data)
+        FileCheck().check("aten::quantize_linear") \
+                   .check_next("aten::int_repr") \
+                   .check_next("aten::_dequantize_linear") \
+                   .check("aten::quantize_linear") \
+                   .check_next("aten::int_repr") \
+                   .check_next("aten::_dequantize_linear") \
+                   .check("aten::quantize_linear") \
+                   .check_next("aten::int_repr") \
+                   .check_next("aten::_dequantize_linear") \
+                   .check("aten::conv2d") \
```

Comment:
This will break after inlining is disabled by default. I suggest porting the test to the final state where it's disabled - I think you would have `prim::CallMethod` nodes here instead of direct aten calls. You could add `@_tmp_donotuse_dont_inline_everything` decorator to the test to disable it for now. Also if you do this there would be no need in running constant-prop.

Reply:
I'll change to final state, since I need this to test qconfig_dict as well

Reply:
it says don't use in the name :), so I won't use it :)

Reply:
oh will probably disable and re-enable later

Reply:
I think it's fine to use it despite the name. And it'll actually be easier to land the "disable-inlining-by-default" PR if we use it because it's easier to just remove decorators than to fix tests that start to fail.